docs/version-history/feature-versions-23-x.rst

```diff
@@ -17,6 +17,11 @@ Release Notes:
 
 New Features:
 
+- Linux systems running cgroup v2 will now hide GPUs that have
+  not been provisioned to the slots (usually because they did not
+  request_gpus.
```

Comment:
```suggestion
  request GPUs).
```

or

```suggestion
  :subcom:`request_gpus`).
```

-- hatever the appropriate :tag: is --

or switch the opening parenthesis to comma and move it left one space.